src/utils/error.js

```diff
@@ -0,0 +1,11 @@
+class CustomError extends Error {
+  constructor({ status, message }) {
+    super()
+    this.message = message || 'An Error occured'
+    this.status = status || 500
+  }
+}
+
+export {
+  CustomError,
+}
```

Comment:
Specify `CustomError`.
Like `InternalServerError` or `NotFoundError` ...etc,
otherwise, I think `new Error()` is pretty enough.
And maybe you would need [es6-error](https://github.com/bjyoungblood/es6-error)